dlp/snippets/deid_test.py

```diff
@@ -299,6 +299,32 @@ def test_deidentify_with_replace_infotype(capsys):
     assert "My favorite site is [URL]" in out
 
 
+def test_deidentify_with_simple_word_list(capsys):
+    deid.deidentify_with_simple_word_list(
+        GCLOUD_PROJECT,
+        "My Room is rm-green",
+        "CUSTOM_ROOM_ID",
+        ["RM-GREEN", "RM-YELLOW", "RM_ORANGE"],
+    )
+
+    out, _ = capsys.readouterr()
+
+    assert "My Room is [CUSTOM_ROOM_ID]" in out
```

Comment:
The output will be "Patient was seen in [CUSTOM_ROOM_ID] then transferred to [CUSTOM_ROOM_ID]"

Reply:
updated